docs/regression_test_api.rst

```diff
@@ -87,18 +88,20 @@ In essence, these builtins exert control over the test creation, and they allow
   One of the most powerful features about these built-in functions is that they store their input information at the class level.
```

Comment:
```suggestion
  One of the most powerful features of these built-in functions is that they store their input information at the class level.
```
I know you didn't update this line, but reading it now, it sounds a bit strange to me with "about".

Reply:
Nice catch! makes total sense to me 👍